### DISCUSSION TITLE
Join our GitHub translation team to help with translation OM into your language

### DISCUSSION THREAD

**Discussion by biodranik** [organicmaps/organicmaps#8538] [Translations]
You can join our [GitHub translation teams](https://github.com/orgs/organicmaps/teams/translations/teams) (visible only to GitHub OM org members), so any contributor can tag all teams or a specific language team in a Pull Request to get help with the review.

Please respond here or let us know which language(s) you can proofread at hello@organicmaps.app if you are interested.

**Comment by ghost** [comment]
I can proofread French language ^^.

**Comment by soshial** [comment]
Polish, Russian, Latvian.

**Reply by biodranik** [reply]
Note that there are no Latvian translations yet, only Lithuanian. It would be great to add a missing translation, more details here: https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md

**Reply by soshial** [reply]
Unfortunately, I don't have experience of initial setup for a language. If you could do automatic DeepL + Google Translate fallback for all strings. Then we could go over it and fix it.

**Reply by biodranik** [reply]
You can take the latest .txt files from the master, and edit them in any text editor. Writing a script that will properly insert automated translations for a _new_ language is an interesting idea btw. @pastk WDYT?

**Reply by pastk** [reply]
Weblate has this feature https://docs.weblate.org/en/latest/user/translating.html#automatic-translation, but we don't have Weblate yet :)

**Reply by biodranik** [reply]
Please stop pushing something that does not solve all our issues easily, and may create other issues instead. There is a whole separate discussion about it.

It should be easy to bootstrap a script that will inject new translations using existing scripts in tools/python folder. I agree that reviewing translations is way easier than typing them from scratch.

**Comment by soshial** [comment]
Also, the teams link you provided returns 404 for me.

**Reply by biodranik** [reply]
Looks like Github allows to see teams only for members of the org.

**Reply by ghost** [reply]
I don't see it. Have I been added to the team?

**Reply by biodranik** [reply]
Invitations have been sent to you. Please check the email.

**Reply by ghost** [reply]
My bad, thanks ^^

**Comment by Hund** [comment]
I would like to help out with the Swedish translations

**Reply by HuggeK** [reply]
I can chip in too!

**Reply by biodranik** [reply]
Thanks! Invited.

**Reply by Hund** [reply]
I spent some time with the translation today. I knew that GitHub is complicated to use, but I didn't know the whole process would be this bad.

Anyway. I did some translation work, and then submitted my changes. And now they're, err.. all gone.

I don't know how, or why. I'm not new to translating things, but I'm new to using GitHub for more than the basics. This has been by far the worst experience of them all.

Why can't you use a good service like Weblate? I'm sure that it would attract a lot more translators, if you used a user-friendly translation tool like Weblate.

**Reply by biodranik** [reply]
What exactly did you do? Can you share the edited file/its zip archive here? If you don't know GitHub or git, that's okay. There are other ways to contribute.

The weblate topic has been already discussed a million times in many places.

**Reply by Hund** [reply]
1. I went through the 999 steps to find the correct file.
2. I chose to edit the file by clicking on the pencil.
3. Which then lets me edit the file. I can also see this message:

> You’re making changes in a project you don’t have write access to. Submitting a change will write it to a new branch in your fork Hund/organicmaps, so you can send a pull request.

4. When I was done, I clicked on the green button "Commit changes...".
5. I then clicked on green button "Sign off and propose changes" in the popup dialog.
6. My changes are now nowhere to be found.

**Reply by biodranik** [reply]
@Hund thanks for the details. Here are your changes: https://github.com/organicmaps/organicmaps/compare/master...Hund:organicmaps:patch-1

You can make a pull request now.

**Comment by Ithink-Icanhelp** [comment]
Lithuanian

**Comment by Maschga** [comment]
I can help with German, if needed.

**Comment by ttrasuns** [comment]
Hello!
I basically have Latvian translation of strings.txt on my computer that I did in last two days and I would be interested in knowing what I should (or could) do next as I don't have experience in working with git. I'm currently not a member of a Latvian OM translation team (if there is any). If there is already some work started for Latvian localization, I can either proofread ir or or just give away the text file.

**Reply by biodranik** [reply]
Awesome, thanks! You can zip and attach the file here, I'll integrate it. I've sent you an invite into lv translations team.

**Reply by biodranik** [reply]
Note that there are other files to translate, before enabling the full language support in the app...

**Reply by ttrasuns** [reply]
[strings.zip](https://github.com/user-attachments/files/16849818/strings.zip)
Thanks!
I'm uploading zipped 'strings.txt'.
It just means that I definitely need to get those other mandatory (main) files translated as well.

**Reply by biodranik** [reply]
Integrated: https://github.com/organicmaps/organicmaps/pull/9486

**Comment by andergd** [comment]
Greetings, I can help with Spanish (European) and Basque

**Comment by unguest** [comment]
Hi everyone ! French overthere 🇫🇷 !

**Comment by dobridabar** [comment]
Hi, I can help with Serbian translations.

**Comment by IdeallyGrey** [comment]
Hello, I can work on Lithuanian translations

**Reply by pastk** [reply]
@rtsisyk could you please add new translators to the relevant teams?

**Comment by hemanggs** [comment]
Glad to help with Hindi Translations

**Comment by kails-ozols** [comment]
Hi
I can help with Latgalian or Latvian translations.

**Reply by soshial** [reply]
Lieliski! Tulkot te ir nedaudz čakarīgi, bet labprāt palīdzēšu, ja būs jautājumi.

**Reply by soshial** [reply]
Mes esam migrejusi uz Weblate riku un tagad tulkot ir pavisam vienkarsi! Vai jus esat gatavi iesaistities? :)
